player/player.view.tree

```diff
@@ -0,0 +1,95 @@
+$mpds_cifplayer_player $mol_view
+	data? null
+	atom_radius_scale 0.6
+	zoom_scale_step 0.3
+	vibrate? null
+	unvibrate null
+	auto /
+		<= dir_light null
+		<= ambient_light null
+		<= atom_box null
+		<= overlay_box null
+		<= cell_box null
+		<= axes_box null
+		<= overlay_changed null
+	sub /
+		<= Three $mpds_cifplayer_lib_three_view
+			scene => scene
+			controls => controls
+			controls_target <= controls_target null
+			camera => camera
+			on_render <= on_render null
+		<= Left_panel $mol_view
+			sub <= left_panel /
+				<= Info $mol_view
+					sub <= info_sub /
+						<= Descr_a $mol_paragraph
+							title <= descr_a \a=
+						<= Descr_b $mol_paragraph
+							title <= descr_b \b=
+						<= Descr_c $mol_paragraph
+							title <= descr_c \c=
+						<= Descr_alpha $mol_paragraph
+							title <= descr_alpha \α=
+						<= Descr_beta $mol_paragraph
+							title <= descr_beta \β=
+						<= Descr_gamma $mol_paragraph
+							title <= descr_gamma \γ=
+				<= Symlabel $mol_pick
+					trigger_content /
+						<= Sym_icon $mol_icon_eye_check
+						<= symlabel \SG
+					bubble_content /
+						<= Toogle_all $mol_button_minor
+							title <= Toogle_all_title \Enable all
+							click? <=> toogle_all_symmetry? null
+						<= Sym_list $mol_scroll
+							sub /
+								<= Sym_checks $mol_list 
+									sub <= sym_checks /
+										<= Sym_check*0 $mol_check_box
+											title <= sym_name* \
+											checked? <=> symmetry_visible*? false
+				<= Center $mol_check_icon
+					checked? <=> centered? true
+					Icon <= Center_icon $mol_icon_image_filter_center_focus
+		<= Tools $mol_view
+			sub /
+				<= Fullscreen $mol_check
+					checked? <=> fullscreen? false
+					Icon <= Expand_icon $mol_icon_arrow_expand_all
+				<= Zoom_section $mol_list
+					sub /
+						<= Zoom_up $mol_button_minor
+							click? <=> zoom_up? null
+							sub / <= Zoom_up_icon $mol_icon_magnify_plus_outline
+						<= Zoom_down $mol_button_minor
+							click? <=> zoom_down? null
+							sub / <= Zoom_down_icon $mol_icon_magnify_minus_outline
+		<= Overlays $mol_view
+			sub <= overlays_sub /
+				<= Switch_overlay $mol_switch
+					value? <=> overlay? \S
+					options <= available_overlays *
+						S \elements
```

Comment:
Please, let's also add `\display none` for an empty overlay.

Reply:
ok, added (but I would prefer without it)

Reply:
why? a user might want to disable any captions display to see just atoms and nothing else

Reply:
because it can be disabled by clicking the currently enabled option, so all options will be disabled